Packs/PaloAltoNetworks_IoT/Playbooks/playbook-iot-check-servicenow.yml

```diff
@@ -0,0 +1,147 @@
+id: PANW IoT ServiceNow Tickets Check
```

Comment:
@michalgold for 6.0 and above sounds like we should use the ticket mirroring feature.